web-common/src/main/java/edu/unc/lib/boxc/web/common/services/AccessCopiesService.java

```diff
@@ -88,6 +88,25 @@ public boolean hasViewableFiles(ContentObjectRecord briefObj, AccessGroupSet pri
         return resp.getResultCount() > 0;
     }
 
+    public ContentObjectRecord getStreamingChildren(ContentObjectRecord briefObj, AccessGroupSet principals) {
```

Comment:
this only returns one record, so it should probably be `getStreamingChild` or `getFirstStreamingChild`